examples/tutorials/Druggablity_assessment_with_Fpocket.ipynb

```diff
@@ -0,0 +1,2112 @@
+{
```

Comment:
<p>Typos:</p><ol><li>Missing spaces after periods in the second paragraph.</li></ol><p>Suggestion:</p><p>"To learn more about binding site <a href="https://github.com/deepchem/deepchem/blob/master/examples/tutorials/Introduction_to_Binding_Sites.ipynb" target="_blank">here's</a> another DeepChem tutorial on binding site." --&gt; "To learn more about binding sites, check out this additional DeepChem tutorial on the topic."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>